### DISCUSSION TITLE
How to display the Server Browser

### DISCUSSION THREAD

**Discussion by GlennGundermann** [codefori/vscode-ibmi#724] [IBM i] [answered]
Hi there,

is there a way to display the Server Browser without disconnecting from your current connection?

Thank you.

Glenn

**Comment by worksofliam** [comment, marked as answer]
@GlennGundermann You have two options:

1. Open a new VS Code window (Control+Shift+N)
2. Disconnect from the current connection

**Reply by GlennGundermann** [reply]
OMG.  Learning Ctrl+Shift+N has made my day.  Thank you so much!!
